cmd/bnscli/cmd_msgfee.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/x/msgfee"
+)
+
+func msgfeeConf(nodeUrl string, msgPath string) (*coin.Coin, error) {
+	queryUrl := nodeUrl + "/abci_query?path=%22/%22&data=%22msgfee:" + url.QueryEscape(msgPath) + "%22"
```

Comment:
You really don't want to use tendermint client, right?
It seems a bit much to create these queries entirely by hand, but it works and if what you like, fine.

Reply:
In my opinion, tendermint client is not good enough. Using plain HTTP has for me the advantage that anyone read and understands it instantly. It is a basic HTTP request. Mocking is straightforward as well.

I would consider using a client when request/response goes beyond simple HTTP and even then I would honestly consider writing a custom client :see_no_evil: